code/mob.dm

```diff
@@ -1049,6 +1049,20 @@
 		LI += W
 	.= LI
 
+/mob/living/critter/get_unequippable()
+	var/list/obj/item/LI = list()
+
+	for (var/obj/item/W in src)
+		if (src.organHolder)
+			if (istype(W, /obj/item/organ/brain) && src.organHolder.brain == W)
```

Comment:
i don't think they should be unequipping any of their organs....

Reply:
Thing is I don't know of any mob critters having any other organs.

However, turns out organholder & implant var stuff is defined on `mob/living` so they don't seem to actually need seperate procs from one another.